test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -488,23 +490,18 @@ class test_ucp_sockaddr : public ucp_test {
     }
 
     static void err_handler_cb(void *arg, ucp_ep_h ep, ucs_status_t status) {
-        test_ucp_sockaddr *self = reinterpret_cast<test_ucp_sockaddr*>(arg);
-        ucp_test::err_handler_cb(static_cast<ucp_test *>(self), ep, status);
-
-        if (status == UCS_ERR_REJECTED) {
-            entity *e = self->get_entity_by_ep(ep);
-            if (e != NULL) {
-                e->inc_rejected_cntr();
-                return;
-            }
-        }
+        ucp_test::err_handler_cb(arg, ep, status);
 
         /* The current expected errors are only from the err_handle test
          * and from transports where the worker address is too long but ud/ud_x
          * are not present, or ud/ud_x are present but their addresses are too
          * long as well */
-        if (status != UCS_ERR_UNREACHABLE) {
-            UCS_TEST_ABORT("Error: " << ucs_status_string(status));
+        switch (status) {
+            case UCS_ERR_REJECTED:
```

Comment:
if (status != REJECTED && status != UNREACHABLE) ABORT

Reply:
why? IMO column is easier readable and editable vs row with brackets

Reply:
switch/case is longer
but anyway, need to fix indentation